src/components/OutsideNavbar.tsx

```
@@ -128,12 +146,24 @@ const OutsideNavbar: React.FC = () => {
+                  onDashboardClick();
+                  handleNavItemClick();
+                }}
+                sx={{ width: '100%' }}
+              >
+                {item.label}
+              </Button>
+            ) : (
+              <SubmenuDropdown
+                label={item.label}
+                path={item.path}
+                children={item.children}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)